src/backend/distributed/metadata/node_metadata.c

```diff
@@ -1341,6 +1445,65 @@ RemoveNodeFromCluster(char *nodeName, int32 nodePort)
 }
 
 
+/*
+ * ErrorIfNodeContainsNonRemovablePlacements throws an error if the input node
+ * contains at least one placement on the node that is the last active
+ * placement.
+ */
+static void
+ErrorIfNodeContainsNonRemovablePlacements(WorkerNode *workerNode)
+{
+	int32 groupId = workerNode->groupId;
+	List *shardPlacements = AllShardPlacementsOnNodeGroup(groupId);
+	GroupShardPlacement *placement = NULL;
+	foreach_ptr(placement, shardPlacements)
+	{
+		if (!PlacementHasHealtyPlacementOnAnotherGroup(placement))
+		{
+			Oid relationId = RelationIdForShard(placement->shardId);
+			char *qualifiedRelationName = generate_qualified_relation_name(relationId);
+
+			ereport(ERROR, (errmsg("cannot remove or disable the node "
+								   "%s:%d because there are some Citus tables "
+								   "with the only healthy shard placements on "
```

Comment:
maybe avoid "healthy" in this error, since we don't have a healthy vs. unhealthy concept